app/src/pages/PageJoinNetwork.re

```diff
@@ -0,0 +1,35 @@
+open Atom;
+open Molecule;
+open Layout;
+open DesignSystem;
+
+module Styles = {
+  open Css;
+
+  let buttonContainer = style([display(`flex), justifyContent(flexEnd)]);
+};
+
+[@react.component]
+let make = (~closeButtonCallback) =>
+  <Modal closeButtonCallback>
```

Comment:
Modal as a render target should not be known to the page. Similar to how the App controls pages are rendered into the section between topbar and footer.  If the Modal is moved to the caller side then we have full flexibility as to where to render this page.

Reply:
That makes sense.